### PR TITLE
feat: better logs

### PR DESCRIPTION
**Pull request by Eddort**
https://github.com/lidofinance/validator-ejector/issues/35

### CODE REVIEW

File: src/services/messages-processor/service.ts [hunk 1]
```diff
@@ -217,7 +217,10 @@ export const makeMessagesProcessor = ({
 
     try {
       await consensusApi.exitRequest(message)
-      logger.info('Message sent successfully to exit', pubKey)
+      logger.info('Message sent successfully to exit', {
```
Comment (avsetsin) [inline src/services/messages-processor/service.ts:220]: maybe it's better `Voluntary exit message sent successfully to Consensus Layer`

File: src/services/messages-processor/service.ts [hunk 2]
```diff
@@ -235,30 +238,33 @@ export const makeMessagesProcessor = ({
...
     messages: ExitMessage[]
   }) => {
-    logger.info('Job started')
+    logger.info('Job started', { OPERATOR_ID: config.OPERATOR_ID })
 
     const toBlock = await executionApi.latestBlockNumber()
     const fromBlock = toBlock - eventsNumber
-    logger.info(`Latest block is ${toBlock}`)
 
+    logger.info(`Latest block is ${toBlock}`)
```
Comment (avsetsin) [inline src/services/messages-processor/service.ts:246]: added a few thoughts about logging here in the separate PR: https://github.com/lidofinance/validator-ejector/pull/37

File: src/services/messages-processor/service.ts [hunk 3]
```diff
@@ -235,30 +238,33 @@ export const makeMessagesProcessor = ({
     eventsNumber: number
     messages: ExitMessage[]
   }) => {
-    logger.info('Job started')
+    logger.info('Job started', { OPERATOR_ID: config.OPERATOR_ID })
```
Comment (kolyasapphire) [inline src/services/messages-processor/service.ts:241]: Better keep the {} keys in same style eg `operatorId` here

File: src/services/messages-processor/service.ts (hunk 2 above)
Comment (kolyasapphire) [reply src/services/messages-processor/service.ts:246]: Moving to here:
```
logger.info(`Fetched the latest block from EL`, { latestBlock: toBlock })
```

File: src/services/messages-processor/service.ts [hunk 4]
```diff
@@ -235,30 +238,33 @@ export const makeMessagesProcessor = ({
...
 
     const toBlock = await executionApi.latestBlockNumber()
     const fromBlock = toBlock - eventsNumber
-    logger.info(`Latest block is ${toBlock}`)
 
+    logger.info(`Latest block is ${toBlock}`)
     logger.info(
       `Fetching events for ${eventsNumber} last blocks (${fromBlock}-${toBlock})`
```
Comment (kolyasapphire) [inline src/services/messages-processor/service.ts:248]:
```
logger.info(`Fetching request events from the Exit Bus`, { eventsNumber, fromBlock, toBlock } )
```

File: src/services/messages-processor/service.ts (hunk 3 above)
Comment (kolyasapphire) [reply src/services/messages-processor/service.ts:241]: Also need staking module id here

**Review by kolyasapphire — APPROVED** [review]

**Pull request merged**